docs/advanced-topics/wes/wes-agc-tutorial.rst

```diff
@@ -0,0 +1,329 @@
+--------------------------
+The Dockstore entry associated with this workflow can be found here `agc-fastq-read-counts <https://dockstore.org/workflows/github.com/dockstore-testing/wes-testing/agc-fastq-read-counts:main?tab=info>`_.
+
+This WDL workflow tabulates read counts of the input fastq file.
+
+    *Dockstore.wdl*
+
+    .. code:: text
+
+        version 1.0
+
+        workflow fastqReadCounts {
+
+            call countFastqReads
+
+            output {
+                File totalReadsFile = countFastqReads.totalReadsFile
+            }
+        }
+
+
+
+        task countFastqReads {
+
+            input {
+                Array[File] inputFastq
+
+                Int memSizeGB = 4
+                Int diskSizeGB = 128
+                String dockerImage = "biocontainers/samtools:v1.9-4-deb_cv1"
+            }
+
+            command <<<
+
+                set -o pipefail
+                set -e
+                set -u
+                set -o xtrace
+
+                READ_COUNT=0
+
+                for fq in ~{sep=' ' inputFastq}
+                do
+                      FILE_COUNT=$(zcat "${fq}" | wc -l )/4
+                      READ_COUNT=$(( $READ_COUNT + $FILE_COUNT ))
+                done
+
+                echo $READ_COUNT > total_reads.txt
+            >>>
+
+            output {
+
+                File totalReadsFile  = "total_reads.txt"
+            }
+
+            runtime {
+                memory: memSizeGB + " GB"
+                disks: "local-disk " + diskSizeGB + " SSD"
+                docker: dockerImage
+                preemptible: 1
+            }
+        }
+
+1. This workflow takes an array of files as an input. Create a file named ``input.json`` in your working directory with contents:
+
+    *input.json*
+
+    .. code:: text
+
+        {
+            "fastqReadCounts.countFastqReads.inputFastq": ["s3://human-pangenomics/working/HPRC_PLUS/HG005/raw_data/Illumina/child/5A1-24481579/5A1_S5_L001_R1_001.fastq.gz"]
+        }
+
+2. As a requirement of AGC input parsing, create a second file named ``agcInputs.json`` in your working directory.
+   This file indicates which WES attachment should be used as the input JSON for the workflow execution step, in this case, we want ``input.json`` to be our input file:
+
+    *agcInputs.json*
+
+    .. code:: text
+
+        {
+            "workflowInputs": "input.json"
+        }
+
+3. Since this workflow is publicly posted on `Dockstore.org <https://dockstore.org/>`_, we can quickly launch it by passing the Dockstore CLI the entry name, input JSON and attachment file:
+
+    .. code:: text
+
+        dockstore workflow wes launch --entry github.com/dockstore-testing/wes-testing/agc-fastq-read-counts:main --json agcInputs.json -a input.json
+
+
+4. The above command will return a unique run ID, similar to:
+
+    .. code:: text
+
+        b4e86806-2dc0-4d70-b494-52651e9b3de0
+
+    Copy the run ID and run the following to get the workflow run logs:
```

Comment:
Don't we generate the full commands, so can't you copy that instead of just the id?

Reply:
By default, only the run ID is printed. If the user adds the `--verbose` flag, then extra details are printed (including the commands to get status/logs). I decided to keep the commands as simple as possible for the tutorial, and so skipped the additional flag. Feel free to push back if there is any disagreement.